packages/trader/src/App/Components/Elements/PositionsDrawer/positions-modal-card.tsx

```diff
@@ -436,8 +290,10 @@ const PositionsModalCard = observer(
                     <React.Fragment>
                         <BinaryLink
                             onClick={togglePositions}
-                            className={classNames('positions-modal-card', {
+                            className={classNames('positions-modal-card', 'dc-contract-card', {
                                 'positions-modal-card--multiplier': is_multiplier,
+                                'dc-contract-card--green': profit_loss > 0 && !is_multiplier,
```

Comment:
Removed `!result` from gradient check as it should be applied for ongoing and closed contracts. Also and `!is_multiplier` as I'm not changing this contract type in this PR.